[2023-05-05 01:06:29,288] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_to_localfile.create_output_file_customers manual__2023-05-05T01:06:26.149563+00:00 [queued]>
[2023-05-05 01:06:29,296] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: postgres_to_localfile.create_output_file_customers manual__2023-05-05T01:06:26.149563+00:00 [queued]>
[2023-05-05 01:06:29,297] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:06:29,297] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2023-05-05 01:06:29,298] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2023-05-05 01:06:29,311] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): create_output_file_customers> on 2023-05-05 01:06:26.149563+00:00
[2023-05-05 01:06:29,319] {standard_task_runner.py:52} INFO - Started process 3921 to run task
[2023-05-05 01:06:29,322] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'postgres_to_localfile', 'create_output_file_customers', 'manual__2023-05-05T01:06:26.149563+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/load_from_postgres.py', '--cfg-path', '/tmp/tmpr5evjb6w', '--error-file', '/tmp/tmpckhb2vp7']
[2023-05-05 01:06:29,327] {standard_task_runner.py:77} INFO - Job 42: Subtask create_output_file_customers
[2023-05-05 01:06:29,431] {logging_mixin.py:109} INFO - Running <TaskInstance: postgres_to_localfile.create_output_file_customers manual__2023-05-05T01:06:26.149563+00:00 [running]> on host ebd7ac88a2d5
[2023-05-05 01:06:29,482] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:06:29,504] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:06:29,511] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2023-05-05 01:06:29,523] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/renderedtifields.py:163 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:06:29,529] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=postgres_to_localfile
AIRFLOW_CTX_TASK_ID=create_output_file_customers
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T01:06:26.149563+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-05T01:06:26.149563+00:00
[2023-05-05 01:06:29,531] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-05-05 01:06:29,532] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'touch /app/datalake/bronze/load_from_postgres/customers 20230505010629']
[2023-05-05 01:06:29,542] {subprocess.py:85} INFO - Output:
[2023-05-05 01:06:29,546] {subprocess.py:93} INFO - Command exited with return code 0
[2023-05-05 01:06:29,559] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/xcom.py:104 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2023-05-05 01:06:29,614] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=postgres_to_localfile, task_id=create_output_file_customers, execution_date=20230505T010626, start_date=20230505T010629, end_date=20230505T010629
[2023-05-05 01:06:29,660] {local_task_job.py:154} INFO - Task exited with return code 0
[2023-05-05 01:06:29,708] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-05 01:06:29,710] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2023-05-05 01:06:29,711] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.7/site-packages/***/models/dag.py:1076 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
